build/Targets/RepoToolset/Version.props

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
```

Comment:
❓ Is the goal to delete this file eventually, but for now switch to an identical copy of the one that we're eventually going to switch to? If so, can you make a note at the top of this file that it should not be changed or modified for use in Roslyn?

Reply:
Yes, that's the intention for all files in RepoToolset. They are not quite identical copies, just similar enough to make the transition easier. I'll diff them when we are ready for the transition.